doc/guides/west/repo-tool.rst

```diff
@@ -257,6 +257,27 @@ If you would like ``west update`` to keep local branches checked out as
 long as they point to commits that are descendants of the new
 ``manifest-rev``, use the ``-k`` (``--keep-descendants``) option.
 
+.. note::
+
+   ``west --rebase`` will fail in projects that have git conflicts
+   between your branch and new commits brought in by the manifest. You
+   should immediately resolve these conflicts as you usually do with
+   ``git``, or you can use ``git -C <project_path> rebase --abort`` to
+   ignore incoming changes for the moment.
+
+   A plain ``west update`` never fails because it does not trying to hold on
```

Comment:
Not quite: `west update` will fail if there are uncommitted changes in the working tree which conflict with the new HEAD.

Reply:
Fair. How about:

```rst
With a clean working tree, a plain ``west update`` never fails [...]
```

Reply:
Done